modular_citadel/code/modules/mob/living/carbon/human/species_types/jellypeople.dm

```diff
@@ -194,5 +194,65 @@
 			if(new_taur != "None")
 				H.dna.features["mam_tail"] = "None"
 		H.update_body()
+
+	else if (select_alteration == "Penis")
+		for(var/obj/item/organ/genital/penis/X in H.internal_organs)
+			qdel(X)
+		var/new_shape
+		new_shape = input(owner, "Choose your character's dong", "Genital Alteration") as null|anything in GLOB.cock_shapes_list
+		if(new_shape)
+			H.dna.features["cock_shape"] = new_shape
+		H.update_genitals()
+		H.give_balls()
+		H.give_penis()
```

Comment:
This gives genitals even if you select "cancel", which is interpreted as null. This means that that the ability cannot be used to remove your genitals. Add a way to remove genitals via these prompts. Ditto for all of the other genital prompts here

Reply:
Addition/removal is handled by the genitals option above, this was just to allow people to orgasm with said new bits.